desktop/app/menu-bar.js

```diff
@@ -27,23 +35,36 @@ export default function () {
     frame: false,
     resizable: false,
     preloadWindow: true,
-    icon: icon,
+    icon: getIcon(),
     // Without this flag set, menubar will hide the dock icon when the app
     // ready event fires. We manage the dock icon ourselves, so this flag
     // prevents menubar from changing the state.
     'show-dock-icon': true,
   })
 
+  const updateIcon = () => {
+    mb.tray.setImage(getIcon())
+  }
+
+  if (process.platform === 'darwin' && systemPreferences && systemPreferences.subscribeNotification) {
+    systemPreferences.subscribeNotification('AppleInterfaceThemeChangedNotification', () => {
```

Comment:
i kinda prefer this as we're ignoring variables that come in and i don't want that passed to the inner closure. if someone mutates that code that could be surprsing